dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ApplicationConfig.java

```diff
@@ -147,6 +148,11 @@
      */
     private String shutwait;
 
+    /**
+     * when in initialization process,throw {@link org.apache.dubbo.common.extension.DubboInitCheckException},stop program
+     */
+    private Boolean check;

```

Comment:
I think `dubbo.application.check` is a little too ambiguous.

Would `extension-loader.duplicate.check` be better? And I also think there's no necessity to declare it in the main trunk, users who want to change the default value can declare in `parameters`.
1. XML
```xml
<dubbo:application>
   <dubbo:parameter key="dubbo.application.extension-loader.duplicate.check" value=true />
</dubbo:application>
```

2. properties file
```properties
dubbo.application.parameters.extension-loader.duplicate.check=true
```

Reply:
Maybe our original intention is different. I assume a scene: when I accidentally customize a dubbo internal filter（just the same name） in another framework, when user use these two jars(our framework & dubbo), how can you guarantee who is loaded first. According to dubbo currently, the first filter loaded will enter the filter chain, and the later loaded same name filter will throw an exception, but this exception is eaten, how do you notify the user. At least we have encountered this situation in Win10 and Centos7, and there is no exception, but the filter does not work.